lib/datapacksjob.js

```diff
@@ -3126,12 +3126,12 @@ DataPacksJob.prototype.downloadPerformanceData = async function (jobInfo) {
 
                         element_data.Username = item.CreatedBy.Username;
                         element_data.UserProfile = item.CreatedBy.Profile.Name;
-
-                        if (elements[key]) {                        
+                       
+                        if (elements[key]) {                      
                             let propertyset = JSON.parse(elements[key][self.vlocity.namespace + '__PropertySet__c']);
                             var element_type = elements[key][self.vlocity.namespace + '__Type__c'];
 
-                            keys.map(k => {            
+                            keys.map(k => {       
```

Comment:
map should be used when performing action on each element otherwise foreach for normal iteration

Reply:
actually there is no change in this  function(downloadPerformanceData) , its just space diff